Integrations/RemoteFileManagement/RemoteFileManagement.py

```diff
@@ -0,0 +1,522 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+'''IMPORTS'''
+
+import tempfile
+import subprocess
+import shutil
+import os
+
+''' GLOBALS '''
+
+AUTHENTICATION = demisto.params().get('Authentication')
+
+HOSTNAME = demisto.params().get('hostname')
+USERNAME = AUTHENTICATION.get('identifier')
+PORT = str(demisto.params().get('port')) if demisto.params().get('port', None) and len(
+    demisto.params().get('port')) > 0 else None
+PASSWORD = AUTHENTICATION.get('password', None)
+CERTIFICATE = AUTHENTICATION.get('credentials').get(
+    'sshkey') if 'credentials' in AUTHENTICATION and 'sshkey' in AUTHENTICATION.get('credentials') and len(
+    AUTHENTICATION.get('credentials').get('sshkey')) > 0 else None
+
+SSH_EXTRA_PARAMS = demisto.params().get('ssh_extra_params').split() if demisto.params().get('ssh_extra_params',
+                                                                                            None) else None
+SCP_EXTRA_PARAMS = demisto.params().get('scp_extra_params').split() if demisto.params().get('scp_extra_params',
+                                                                                            None) else None
+DOCUMENT_ROOT = '/' + demisto.params().get('document_root') if demisto.params().get('document_root', None) else None
+
+''' UTILS '''
+
+
+def create_certificate_file(password: str, certificate: str):
+    cert_file = tempfile.NamedTemporaryFile()
+    if certificate:
+        with open(cert_file.name, "w") as f:
```

Comment:
No need to open. NamedTemporaryFile() will return an open file. Also make sure to flush after write.